src/js/components/RegisterPage/SignUp.sass

```diff
@@ -1,6 +1,8 @@
-.SignUp
-    margin: 0 auto
-    margin-top: 70px
-    max-width: 400px
-    padding: 40px 40px
-    background: #fff
\ No newline at end of file
+
+.Signup form
+  margin: 0 auto;
+  margin-top: 70px;
+  max-width: 400px;
+  padding: 40px 40px
+  background: #fff
```

Comment:
.Signup form{
  margin: 0 auto;
  margin-top: 70px;
  max-width: 400px;
  padding: 40px 40px;
  background: #fff;
}

Reply:
bro... did you test if its compiling?
theres no ";" after   padding: 40px 40px & background: #fff

Reply:
I commented the same thing, sass apparently does not need semi colons or brackets to compile and uses indentation instead
Reply from above: 
> upon reviewing the sass documentation on this url "https://sass-lang.com/guide"
> It states that sass has no brackets nor semi colon as it is tab indented like Python code could would be.

Reply:
sure... but then the code should be consistent @SamNHarris 
because using ; on some and not on others are bad
`.Signup form
  margin: 0 auto;
  margin-top: 70px;
  max-width: 400px;
  padding: 40px 40px
  background: #fff`

Reply:
Agreed, it should be consistent 👍